api/libproject/devstart.rb

```diff
@@ -578,6 +578,33 @@ def cloudsql_import(cmd_name, *args)
                             :fn => lambda { |*args| cloudsql_import("cloudsql-import", *args) }
                         })
 
+def local_mysql_import(cmd_name, *args)
+  op = WbOptionsParser.new(cmd_name, args)
+
+  op.add_option(
+    "--sql-dump-file [filename]",
+    lambda {|opts, v| opts.file = v},
+    "File name of the SQL dump to import"
+  )
+  op.add_option(
+    "--bucket [bucket]",
+    lambda {|opts, v| opts.bucket = v},
+    "Name of the GCS bucket containing the SQL dump"
+  )
+  op.parse.validate
+
+  common = Common.new
+  common.run_inline %W{docker-compose run db-local-mysql-import
+        --sql-dump-file #{op.opts.file} --bucket #{op.opts.bucket} --project #{op.opts.project}}
```

Comment:
--project isn't used? I don't think it's even set in options here?

Reply:
I thought i removed that everywhere. i'll fix